src/app/booking/OOSPlannerSection.js

```diff
@@ -104,7 +108,11 @@ function FilledBooking({ booking }) {
       <Typography>{booking.activity.address}</Typography>
       <Box sx={{ mt: "auto", mb: -1.5, mx: -1.5, display: "flex", justifyContent: "space-between" }}>
         <IconButton>
-          <IosShareIcon />
+          <IosShareIcon onClick={() => {
```

Comment:
So you pass 2 functions and call them one after another. Why do not pass single function showSharePopup?